keps/sig-storage/1491-csi-migration-vsphere/README.md

```diff
@@ -55,4 +82,7 @@ Major milestones for vSphere in-tree plugin CSI migration:
   - vSphere CSI migration to Beta, off by default
   
 - 1.24
-  - vSphere CSI migration to Beta, on by default
\ No newline at end of file
+  - vSphere CSI migration to Beta, off by default
```

Comment:
This should be 1.24->1.25: off by default -> on by default

Reply:
done